### PR TITLE
add boo-usdc-dei

### PR DESCRIPTION
**Pull request by amatureApe**
Moo Boo USDC-DEI

Vault: 0xa43A80F527aC3619ce904C55758D83A7b468fb16
Strategy: 0x7bDA38F77D424b219F874a6e3f12dA3856828585
Want: 0x91f7120898b4be26cC1e84F421e76725c07d1361
PoolId: 2

I think this vault should already work with the API changes made here - https://github.com/beefyfinance/beefy-api/pull/937

### CODE REVIEW

File: src/config/vault/fantom.json [hunk 1]
```diff
@@ -113,6 +113,36 @@
...
+      "COMPLEXITY_LOW",
+      "BATTLE_TESTED",
+      "IL_LOW",
+      "ALGO_STABLE",
+      "MCAP_MICRO",
+      "AUDIT",
+      "CONTRACTS_VERIFIED"
+    ],
+    "addLiquidityUrl": "https://spooky.fi/#/add/0xDE1E704dae0B4051e80DAbB26ab6ad6c12262DA0/0x04068DA6C83AFCFA0e13ba15A6696662335D5B75",
+    "buyTokenUrl": "https://spooky.fi/#/swap?outputCurrency=0xDE1E704dae0B4051e80DAbB26ab6ad6c12262DA0",
```
Comment (cartman-dev) [inline src/config/vault/fantom.json:142]: You did this right, but you should notice when you test it that spooky.fi does not take inputCurrency and outputCurrency fields as expected.  Not sure why...

**Review by cartman-dev — COMMENTED** [review]
APY is not loading.  Perhaps due to ID's not matching.  Can you confirm with a local test?

File: src/config/vault/fantom.json (hunk 1 above)
Comment (amatureApe) [reply src/config/vault/fantom.json]: Yea spooky is a bit weird. I just put the URL as it is when at the buy page which is https://spooky.fi/#/swap

File: src/config/vault/fantom.json [hunk 2]
```diff
@@ -113,6 +113,36 @@
     "createdAt": 1646162527,
     "mintTokenUrl": "https://beta.beefy.finance/#/fantom/vault/beefy-beFTM"
   },
+  {
+    "id": "boo-dei-usdc-v2",
```
[the remark this answers is not in the document]
Comment (amatureApe) [reply src/config/vault/fantom.json]: Gotcha. Yup this was the issue. Should be resolved now

**Pull request merged**